src/_assets/css/_overwrites.scss

```diff
@@ -295,11 +295,6 @@ body.homepage .get-started-section {
 	.alert, pre {
 		border: 1px solid black;
 	}
-
-	/* Hide "Open Dartpad" link */
-	.run-in-dartpad {
-		display: none;
-	}
```

Comment:
This CSS class was introduced July 11, 2016, [Migrating repo from Ocupop](https://github.com/dart-lang/site-www/commit/fde624e85deceb0590beeb5ab65a06f8e31a1ffa). AFAICT this style isn't used anymore (I've checked the site sources and the generated site).